### PR TITLE
PIC-3488: new tabs based on heard/not heard cases

### PR DESCRIPTION
**Pull request by paddybasi**

### CODE REVIEW

File: server/services/case-service.js [hunk 1]
```diff
@@ -34,9 +34,16 @@ const createCaseService = (apiUrl) => {
       apiUrlBuilder.searchParams.append('VERSION2', 'true')
       apiUrlBuilder.searchParams.append('page', page || '1')
       apiUrlBuilder.searchParams.append('limit', limit)
+
       if (subsection === 'added') {
         apiUrlBuilder.searchParams.append('recentlyAdded', 'true')
       }
+      if (subsection === false || subsection === null || subsection === undefined) {
```
Comment (johnyfarrar) [inline server/services/case-service.js:41]: Does this need a flag? cos what happens if you turn off the outcomes flag?

**Review by johnyfarrar — COMMENTED** [review]
Just one comment for now - until we see the tests passing

File: server/services/case-service.js (hunk 1 above)
Comment (paddybasi) [reply server/services/case-service.js:41]: The `getPagedCaseList` method is already behind a feature flag `ENABLE_SERVERSIDE_PAGING`, but yes I suppose I need to take into account if this flag is enabled but `ENABLE_HEARING_OUTCOMES` isn't.

File: server/services/case-service.js [hunk 2]
```diff
@@ -34,9 +34,16 @@ const createCaseService = (apiUrl) => {
...
+        apiUrlBuilder.searchParams.append('hearingStatus', 'UNHEARD')
```
[the remark this answers is not in the document]
Comment (paddybasi) [reply server/services/case-service.js:42]: Is the `hearingStatus` now a required param to call in CCS? I've written another if (yet to push) around this to look for the outcomes flag

[the remark this answers is not in the document]
Comment (paddybasi) [reply server/services/case-service.js:42]: Reason I'm asking if it's required is you're suggesting of setting it to null

**Review by johnyfarrar — APPROVED** [review]
LGTM

**Review by rschallapalli — APPROVED** [review]
LGTM

**Pull request merged**